index.bs

```diff
@@ -451,24 +451,21 @@ spec: SHA2; urlPrefix: http://csrc.nist.gov/publications/fips/fips180-4/fips-180
   valid metadata as described by the following ABNF grammar:
 
   <pre dfn-type="grammar" link-type="grammar">
-      <dfn>integrity-metadata</dfn> = *<a>WSP</a> <a>hash-with-options</a> *(1*<a>WSP</a> <a>hash-with-options</a> ) *<a>WSP</a> / *<a>WSP</a>
```

Comment:
`WSP` defined in [2.2 Grammatical Concepts](https://www.w3.org/TR/SRI/#grammatical-concepts) referencing [2.4.1 Common parser idioms](http://www.w3.org/TR/html5/infrastructure.html#space-character). The link is broken and redirects to the https://html.spec.whatwg.org and there is no definition about `WSP`. Also, [Terms defined by reference](https://pr-preview.s3.amazonaws.com/baek9/webappsec-subresource-integrity/pull/112.html#index-defined-elsewhere) uses `WSP` with `VCHAR` defined in [RFC5234:ABNF](https://tools.ietf.org/html/rfc5234).

Therefore, in [2.2 Grammatical Concepts](https://www.w3.org/TR/SRI/#grammatical-concepts), it seems that `WSP` should also be modified to use the definition of [RFC5234:ABNF](https://tools.ietf.org/html/rfc5234) along with `VCHAR`.